catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrFilterDelegate.java

```diff
@@ -82,6 +92,21 @@
 
     private static final WKTWriter WKT_WRITER = new WKTWriter();
 
+    private static final Map<String, String> SPATIAL_CONTEXT_ARGUMENTS = ImmutableMap.of(
+            "spatialContextFactory",
+            JtsSpatialContextFactory.class.getName(),
+            "validationRule",
+            ValidationRule.repairConvexHull.name(),
+            "allowMultiOverlap",
```

Comment:
Would be good to comment on these options, as these geospatial tools can be somewhat opaque. There is some discussion on it here: https://github.com/locationtech/spatial4j/issues/34#issuecomment-12426283


Do the unit tests exercise both the `repair` and `MultiOverlap` modifications?